share/workflows/jobs/internal-gui/scripts/csv_export.py

```diff
@@ -2,7 +2,13 @@
 import re
 
 import pandas
-from PyQt4.QtGui import QCheckBox
+import sys
+
+if sys.version_info[0] == 2:
+  from PyQt4.QtGui import QCheckBox
+else:
+  from PyQt5.QtWidgets import QCheckBox
```

Comment:
Instead of explicitly testing Python version I would hope this could be:
```
try:
   from PyQt4.QtGui import QCheckBox
except ImportError:
   from PyQt5.QtWidgets import QCheckBox
```

Reply:
Are we going to Qt5 in 2.3?

Reply:
As I see it this is mainly a change to *prepare* for Python 3; since Python 3 requries Qt5 we have this thing going. I had not thought we would go for Qt5 in 2.3.

Reply:
It might be better to use six here to check python version. It has two flags you can check for, I think they are `six.PY2` and `six.PY3`